docs/L1_Services/LSSTCam_Prompt_Processing_Services.tex

```diff
@@ -0,0 +1,208 @@
+in the same filter with no slew of the telescope. However, a new
+program, potentially with a new filter, a larger slew, a different
+observing cadence, or a different visit structure, can be scheduled at
+any moment.
+
+Another evisioned program is ``deep drilling'', where many more exposures
+than the two exposure visit will be taken.
+
+In practice, science data-taking will proceed when conditions are
+suitable. Calibration data may be taken when conditions are not
+suitable for further observations, with science data-taking resuming
+when conditions again become suitable.
+
+It follows that the desired behavior for science data-taking
+operations is to start the Prompt Processing system at the beginning
+of the night and to turn off the system after all processing for all
+science observations is finished.  
+
+The operational framework for observing discloses no future knowlege
+about what exposures will be taken, until the “next visit” is
+determined.
+
+During science data-taking the Prompt Processing Service computes and
+promptly returns QA parameters (referred to as “telemetry”) to the
+observing system. The QA parameters are not specific to an observing
+program; examples are seeing and pointing corrections derived from the
+WCS. These parameters are not strictly necessary for observing to
+proceed -- LSST can observe autonomously at the telescope site, if
+need be. Also note that the products are quality parameters, not the
+the “up-or-down” quality judgement.
+
+The scheduler may be sensitive to a subset of these messages and may
+decide on actions, but a detailed description is TBD and may vary as
+the survey evolves. The scheduler can make use of these parameters even
+if delivered quite late, since the scheduler uses historical data
+in addition to recent h data.
+
+The Prompt Processing system also executes code that is specific to an
+observing program. For science exposures, the code is divided into
+a front-end -- that is able to compute the parameters sent back  to the
+observator, and a back end , Alert Production (AP), is the specific
+science code that detects transient objects.
+
+The detected transients are passed off to another service, which
+records the data in a catalog which can be queried offline and sends
+events to an ensemble of transient brokers. Data are transmitted to end
+users either via feeds from an LSST-provisioned broker or via
+community-provided alert brokers.
+
+Alert Production runs in the context of the wide-fast-deep survey,
+deep drilling program and TBD other programs. Other observing programs
+may also include AP as a science code, or may have codes of their own.
+
+\subparagraph{Calibration Operations}
+
+In addition to collecting data for science programs, the telescope and
+camera are involved in many calibration activities.
+
+the Baseline Calibrations include flats and biases. Darks are not anticipated.
+LSST has a collumanted Beam projector calibration device as well.

```

Comment:
DMS-REQ-0130 is "The DMS shall produce and archive Calibration Data Products... including at least... Dark correction frames". See also LDM-151 §4.2.4.

(Also, looks like there's some proofreading needed on the above paragraph.)